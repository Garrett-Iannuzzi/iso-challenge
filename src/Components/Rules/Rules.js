import React from 'react';
import './Rules.scss';

const Rules = () => {
  return(
    <article>
      <h2>Iso-Challenge Rules and Regulations</h2>
      <ul>
        <li>Choose a team name & skill level</li>
        <li>Choose a player for each position from the list</li>
        <li><b>Draft style:</b> a player can only appear on one team</li>
        <li>Click submit when both teams are full</li>
        <li><b>Iso-Challenge</b> will unlock all game stats for your players</li>
        <li>Argue away ay which team has the better stats</li>
        <li> &#128170; NO FIGHTING &#128170;</li>
      </ul>
    </article>
  )
}

export default Rules;