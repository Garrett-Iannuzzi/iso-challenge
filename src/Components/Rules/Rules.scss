article {
  border: solid 4px black;
  background-color:rgb(174, 142, 179);
  height: 30em;
  margin: 0 auto;
  margin-top: 30px;
  text-align: center;
  width: 50%;
  &>ul {
    list-style-type: none;
    &>li {
      font-size: 1.4em;
      margin: 1em;
    }
  }
}