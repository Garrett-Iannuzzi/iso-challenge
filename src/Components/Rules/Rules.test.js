import React from 'react';
import Rules from './Rules';
import { shallow } from 'enzyme';

describe('Rules', () => {

  let wrapper

  describe('Rules Component', () => {

    beforeEach(() => {
      wrapper = shallow(<Rules />)
    });

    it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    });
  });

})