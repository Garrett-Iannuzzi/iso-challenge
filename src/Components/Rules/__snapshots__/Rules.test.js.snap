// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rules Rules Component should match the snapshot 1`] = `
<article>
  <h2>
    Iso-Challenge Rules and Regulations
  </h2>
  <ul>
    <li>
      Choose a team name & skill level
    </li>
    <li>
      Choose a player for each position from the list
    </li>
    <li>
      Draft style: a player can only appear on one team
    </li>
    <li>
      Click submit when both teams are full
    </li>
    <li>
      Iso-Challenge will unlock all game stats for your players
    </li>
    <li>
      Argue away ay which team has the better stats
    </li>
    <li>
      NO FIGHTING
    </li>
  </ul>
</article>
`;
